x-pack/docs/en/rest-api/security/saml-authenticate-api.asciidoc

```diff
@@ -0,0 +1,89 @@
+[role="xpack"]
+[[security-api-saml-authenticate]]
+=== SAML authenticate API
+
+Submits a SAML response message to {es} for consumption when using a custom web application other than {kib}.
+If you are using {kib}, please see the {stack-ov}/saml-guide.html[SAML guide]
+
+[[security-api-saml-authenticate-request]]
+==== {api-request-title}
+
+`POST /_security/saml/authenticate`
+
+[[security-api-saml-authenticate-desc]]
+==== {api-description-title}
+
+The SAML response message that is submitted can be:
+
+* a response to a SAML authentication request that was previously created using the
```

Comment:
Maybe it's easier to parse if it's split into two sentences. For example, something like this:
```suggestion
* A response to a SAML authentication request. The response must be generated by the
```

Reply:
It's actually the authentication request that was generated, not the response. This does get too complicated :(